### PR TITLE
Add a status command

### PR DESCRIPTION
**Pull request by nono**

### CODE REVIEW

File: cmd/status.go [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+// statusCmd represents the status command
+var statusCmd = &cobra.Command{
+	Use:   "status",
+	Short: "Check if the HTTP server is running",
+	Long:  `Check if the HTTP server has been started and answer 200 for /status.`,
+	Run: func(cmd *cobra.Command, args []string) {
+		port := os.Getenv("PORT")
+		if len(port) == 0 {
+			port = "8080"
```
Comment (aenario) [inline cmd/status.go]: We should let viper handle the whole config or ENV or default. Can be the object of a separate PR.

File: cmd/status.go [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+		}
+		resp, err := http.Get("http://localhost:" + port + "/status")
+		if err != nil {
+			fmt.Println("Error the HTTP server is not running:", err)
+			os.Exit(1)
```
Comment (aenario) [inline cmd/status.go]: Should we have significant error code ? In which case we should also named constants.

File: cmd/status.go (hunk 1 above)
Comment (nono) [inline cmd/status.go]: Yes, it should be better. For the moment, it's just the default way of how gin works. But later, we should definetively move that in viper.

File: cmd/status.go (hunk 2 above)
Comment (nono) [inline cmd/status.go]: I think we should just have 0 for OK and something else for an error. Error messages are more convenient to know what has failed.
